### PR TITLE
Adding docs for testing Accumulo release

### PR DESCRIPTION
**Pull request by mikewalch**

### CODE REVIEW

File: contributor/contributors-guide.md [hunk 1]
```diff
@@ -570,7 +570,65 @@ For details about governance policies for the Accumulo project view the followin
...
+     </profiles>
+     <activeProfiles>
+       <activeProfile>accumuloRC</activeProfile>
+     </activeProfiles>
+   </settings>
+   EOF
+   ```
+1. If a new release candidate is made, update your staging repo ID.
+   ```shell
+   $ export RC_STAGING=1071
```
Comment (ctubbsii) [inline contributor/contributors-guide.md]: Probably don't need this section. I think it's probably already obvious that the staging number corresponds to the staging repo that you want to test, and should change if you want to test a different staging repo.

File: contributor/contributors-guide.md [hunk 2]
```diff
@@ -570,7 +570,65 @@ For details about governance policies for the Accumulo project view the followin
...
+
+#### Run the integration tests of projects that use Accumulo
+
+1. Clone the [Accumulo Examples] project:
+    ```shell
+    $ git clone https://github.com/apache/accumulo-examples.git
+    ```
+1. Run the integration test
+    ```shell
+    $ mvn clean verify -Daccumulo.version=$RC_VERSION
```
Comment (ctubbsii) [inline contributor/contributors-guide.md]: I'd rather inline the `-s /tmp/accumulo-rc-maven.xml` into these two examples. An alias could easily be forgotten, a user won't be able to use that same shell session for other maven tasks, without possible unexpected behavior.

File: contributor/contributors-guide.md (hunk 1 above)
Comment (mikewalch) [reply contributor/contributors-guide.md]: Removed in c3749ada76db4

File: contributor/contributors-guide.md (hunk 2 above)
Comment (mikewalch) [reply contributor/contributors-guide.md]: Fixed in c3749ada76db4

**Pull request merged**